plugins/module_utils/api.py

```diff
@@ -60,11 +60,11 @@ def transform_query_to_servicenow_query(query):
     )
 
 
-def table_name(module):
+def resource_name(module):
```

Comment:
I would keep `table_name` API for backward compatibility (someone might be using this API in their custom modules).

Add a deprecation note in the function doc.